build.gradle

```diff
@@ -317,17 +317,40 @@ testClusters.integTest {
     setting 'path.repo', repo.absolutePath
 }
 
-String bwcVersion = "2.4.0-SNAPSHOT"
+String baseVersion = "2.5.0"
+String bwcVersion = baseVersion + ".0"
 String baseName = "obsBwcCluster"
 String bwcFilePath = "src/test/resources/bwc/"
-String remoteFileURL = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.4.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-observability-2.4.0.0.zip"
+String bwcObservabilityPlugin = "opensearch-observability-" + bwcVersion + ".zip"
+String bwcJobSchedulerPlugin = "opensearch-job-scheduler-" + bwcVersion + ".zip" 
+String remoteFileURL = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${baseVersion}/latest/linux/x64/tar/builds/opensearch/plugins/" + bwcObservabilityPlugin
+String bwcJobSchedulerURL = "https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${baseVersion}/latest/linux/x64/tar/builds/opensearch/plugins/" + bwcJobSchedulerPlugin
 
 2.times {i ->
     testClusters {
         "${baseName}$i" {
             testDistribution = "ARCHIVE"
-            versions = ["2.4.0",opensearch_version]
+            versions = [baseVersion,opensearch_version]
             numberOfNodes = 3
+            plugin(provider(new Callable<RegularFile>(){
```

Comment:
Unsure about how the CI was working before, but it seems like the job scheduler needs to be included in the bwc part of the CI

Reply:
Otherwise it is failing with the following error:
``` Exception in thread "main" java.lang.IllegalArgumentException: Missing plugin [opensearch-job-scheduler], dependency of [opensearch-observability]```

Reply:
yes - same issue for SQL ...